BrainPortal/app/models/cluster_task.rb

```diff
@@ -2038,6 +2038,7 @@ def docker_commands(command_script)
     run                                          \\
     -u #{Process.uid}:#{Process.gid}             \\
     --rm                                         \\
+    --entrypoint /bin/sh                         \\
```

Comment:
Why `/bin/sh` instead of `/bin/bash` ?!?

Reply:
Not all slimmed down linux containers have `/bin/bash`, all that I've found do have `/bin/sh`. We've had this discussion within Boutiques and was the conclusion we arrived at and has been successful.

Reply:
alpine is an example, it is the canonical docker image for python programs and doesn't have /bin/bash available.  This is what precipitated this same change in boutiques.

Reply:
OK. Anyway. /bin/sh is a buggy interpreter, very limited, it coudl cause serious problems if the script that you invoke was designed for /bin/bash ...